docker-compose.yml

```diff
@@ -6,38 +6,56 @@ services:
         command: >
             /bin/bash -c "
                 until echo > /dev/tcp/postgres/5432; do sleep 1; done
-                npx pubsweet server
+                node app
             "
         ports:
             - 3000:3000
         depends_on:
             - postgres
+        env_file:
+          - .env
         environment:
-            NODE_ENV: development
+            NODE_ENV: production
             NODE_CONFIG: '{
-                "pubsweet-client": {
-                    "API_ENDPOINT": "/api"
-                },
-                "pubsweet-server": {
-                    "baseUrl": "${PUBSWEET_BASEURL}",
-                    "secret": "somesecret",
-                    "hostname": "${PUBSWEET_HOSTNAME}"
-                },
-                "password-reset": {
-                    "url": "${PUBSWEET_BASEURL}/password-reset"
-                },
-                "pubsweet-component-ink-backend": {
-                    "inkEndpoint": "https://demo--ink.elifesciences.org",
-                    "email": "fake",
-                    "password": "fake"
+                "scienceBeam":{
+                    "url":"http://sciencebeam:8075/api/convert"
                 },
                 "auth-orcid": {
-                    "clientID": "${ORCID_CLIENT_ID}",
-                    "clientSecret": "${ORCID_CLIENT_SECRET}"
+                    "sandbox": true
                 }
             }'
             PGHOST: postgres
             PGUSER: xpub
+
+    sciencebeam:
+        image: elifesciences/sciencebeam:${SCIENCEBEAM_TAG}
+        command: ./server.sh \
+            --host=0.0.0.0 --port=8075 \
+            --grobid-url http://grobid:8070/api
+        ports:
+        - "8075:8075"
```

Comment:
this port can be useful to debug or smoke test `sciencebeam` on `localhost:8075`; it will be closed on the server's security group, so not available outside of there

Reply:
I don't understand. Is there any action required here?

Reply:
No, just an heads up that `sciencebeam` will only be accessible from the host itself, and its containers